test/playlist-selectors.test.js

```diff
@@ -121,3 +122,34 @@ function(assert) {
   assert.equal(result.playlist, master.playlists[0], 'selected the correct playlist');
   assert.equal(result.rebufferingImpact, 1, 'impact on rebuffering is 1 second');
 });
+
+test('lowestBitrateCompatibleVariantSelector picks lowest non-audio playlist',
+  function(assert) {
+    // Set this up out of order to make sure that the function sorts all
+    // playlists by bandwidth
+    this.hls.playlists.master.playlists = [
+      { attributes: { BANDWIDTH: 10, CODECS: 'mp4a.40.2' } },
+      { attributes: { BANDWIDTH: 100, CODECS: 'mp4a.40.2, avc1.4d400d' } },
+      { attributes: { BANDWIDTH: 50, CODECS: 'mp4a.40.2, avc1.4d400d' } }
+    ];
+
+    const expectedPlaylist = this.hls.playlists.master.playlists[2];
+    const testPlaylist = lowestBitrateCompatibleVariantSelector.call(this.hls);
+
+    assert.equal(testPlaylist, expectedPlaylist,
+      'Selected lowest compatible playlist with video assets');
+  });
+
+test('lowestBitrateCompatibleVariantSelector picks lowest audio rendition if no video exists',
```

Comment:
The description is no longer accurate (now returns null) Same goes for the assertion description below